packages/medusa/src/api-v2/admin/price-lists/types.ts

```diff
@@ -0,0 +1,25 @@
+import { PriceListStatus, PriceListType } from "@medusajs/types"
+
+export type AdminPriceListEndpointDTO = {
```

Comment:
question: I believe that this type doesn't make real sense without consuming listPriceLists, if that the case then I would move that type in the same file as the function and maybe not calling it a EndPointDTO but maybe just `AdminSanitizedPriceListDTO`?

Reply:
This type is for the remote query collection result, which is also the result of the endpoint, which is why I named it like that. Could be useful for the frontend as well since it's the resultant type of the endpoint. We can definitely name this something else and move it somewhere else.

Wdyt?